www/versions.json

```diff
@@ -0,0 +1,257 @@
+{
+  "std": [
+    "0.120.0",
+    "0.119.0",
+    "0.118.0",
```

Comment:
@lucacasonato are we gonna have to sync this list in both `manual` and `dotland` repos?